### DISCUSSION TITLE
Missing module error in Federated Learning with NVFlare tutorial

### DISCUSSION THREAD

**Discussion by sorensenmarius** [Project-MONAI/tutorials#376] [Q&A]
I have followed each step of the federated learning with NVFlare without Docker tutorial, and have run into a problem when I want to start the server and the clients. When I do the "Run federated learning" step in the README I am getting this error from sub_start.sh:
`/tutorials/federated_learning/nvflare/nvflare_example/nvflare_monai/bin/python3: No module named nvflare.private.fed.app.client.client_train`
and 
`/tutorials/federated_learning/nvflare/nvflare_example/nvflare_monai/bin/python3: No module named nvflare.private.fed.app.server.server_train`

I have tried redoing the whole installation and manually installing nvflare with both pip and pip3, however these tell me that all requirements are satisfied. 

Has anyone run into this problem? I would love some input on how to move forward with this.

**Comment by wyli** [comment]
Thanks for reporting, I created an issue for this https://github.com/Project-MONAI/tutorials/issues/377

**Comment by yiheng-wang-nv** [comment]
Hi @sorensenmarius , could I ask which python version did you used?

**Reply by sorensenmarius** [reply]
Seems to be 3.6.9

**Reply by yiheng-wang-nv** [reply]
Thanks, if possible, could you please try with python 3.8 and check if it works?

**Reply by yiheng-wang-nv** [reply]
Hi @sorensenmarius , I've checked with the released nvflare==1.0.2, and the built pyc files was from python 3.8.10, thus this python version should be used. Thanks for reporting this issue, and later I will submit a PR to modify the readme.

**Reply by sorensenmarius** [reply]
I tried this with Python 3.8.10 and I am still getting the same problem.

**Reply by holgerroth** [reply]
Have you tried using a virtual environment for this? See instructions [here](https://github.com/Project-MONAI/tutorials/tree/master/federated_learning/nvflare/nvflare_example#environment-setup).

**Comment by TaocsZhang** [comment]
I have tried python 3.8.10, nvflare==1.0.2 and monai==0.7.0. I use conda to create the virtual environment. 
Any idea to solve this?

I have the same issue: /tutorials/federated_learning/nvflare/nvflare_example/nvflare_monai/bin/python3: No module named nvflare.private.fed.app.client.client_train
and
/tutorials/federated_learning/nvflare/nvflare_example/nvflare_monai/bin/python3: No module named nvflare.private.fed.app.server.server_train

**Reply by yiheng-wang-nv** [reply]
Hi @TaocsZhang , can you try to follow the instructions in [readme](https://github.com/Project-MONAI/tutorials/blob/master/federated_learning/nvflare/nvflare_example/README.md) and check if it works?
